gitpod-io/gitpod
# [ws-manager-bridge] Use WS update status version

## Description
This PR makes ws-manager-bridge respect and record the status version reported by ws-manager.
Prior this behaviour was only used for prebuilds. We need to extend this behaviour to get rid of a host of heuristics which aim to identify the latest update, and to properly support the public API.

If I understand correctly, this change renders the prebuild status check ineffective because it preempts this check. If this is really the case, we want to consider removing the prebuild status check.

cc @easyCZ 

## How to test
1. Start and stop workspaces - things should work as normal
2. Inspect the database/instance updates. You should see the status.version increment

## Release Notes

```release-note
NONE
```

## Werft options:
